bigquery_magics/bigquery.py

```diff
@@ -97,15 +98,18 @@
 import ast
 from concurrent import futures
 import copy
+import json
 import re
 import sys
+from threading import Thread
```

Comment:
Nit: per https://google.github.io/styleguide/pyguide.html#22-imports "Use import statements for packages and modules only, not for individual types, classes, or functions." with only a few exceptions, such as for the `typing` module's attributes.